src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Interfaces/IThemeService.cs

```diff
@@ -1,11 +1,11 @@
 ﻿using System;
 using System.Collections.Generic;
-using GovUk.Education.ExploreEducationStatistics.Admin.Models;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
 
 namespace GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces
 {
     public interface IThemeService
     {
-        List<Theme> GetThemes(Guid guid);
+        List<Theme> GetByUser(Guid guid);
```

Comment:
I wonder if we should go with something slightly more descriptive like `GetUserThemes` as i suspect we will end up with a bunch more stuff in here

Reply:
Can do